qcfractal/cli/qcfractal_manager.py

```diff
@@ -6,64 +6,56 @@
 
 import qcfractal
 import tornado.log
+import qcengine as qcng
 
 from . import cli_utils
 
 __all__ = ["main"]
 
 
 def parse_args():
-    parser = argparse.ArgumentParser(description='A CLI for the QCFractal QueueManager.')
-    subparsers = parser.add_subparsers(help='QueueManager Backend Type', dest='adapter_type')
-    subparsers.required = True
-
-    # Keywords for Dask
-    dask_parser = subparsers.add_parser('dask', help='Dask')
-    dask_parser.add_argument("--dask-uri", type=str, help="URI of the dask-server")
-    dask_parser.add_argument(
-        "--local-cluster", action="store_true", help="Start a Dask LocalCluster rather than connect to a scheduler")
-    dask_parser.add_argument(
-        "--local-workers", type=int, default=None, help="The number of workers for the LocalCluster")
-
-    # Keywords for Fireworks
-    fw_parser = subparsers.add_parser('fireworks', help='Fireworks')
-    fw_parser.add_argument("--fw-config", type=str, help="A FWConfig file")
-    fw_parser.add_argument("--fw-uri", type=str, help="URI of MongoDB server")
-    fw_parser.add_argument(
-        "--fw-name", type=str, default="qcfractal_fireworks_manager", help="The MongoDB Database to use locally")
+    parser = argparse.ArgumentParser(
+        description='A CLI for a QCFractal QueueManager with a ProcessPoolExecutor backend.')
 
     # Keywords for ProcessPoolExecutor
-    executor_parser = subparsers.add_parser('executor', help='ProcessPoolExecutor')
-    executor_parser.add_argument("--nprocs", type=int, help="The number of process for the executor")
+    executor = parser.add_argument_group('Executor settings')
+    executor.add_argument(
+        "--ntasks",
+        required=True,
+        type=int,
+        help="The number of simultaneous tasks for the executor to run, resources will be divided evenly.")
+    executor.add_argument("--cores", type=int, help="The number of process for the executor")
+    executor.add_argument("--memory", type=int, help="The total amount of memory on the system in GB")
 
     # FractalClient options
-    parser.add_argument(
+    server = parser.add_argument_group('FractalServer connection settings')
+    server.add_argument(
         "--fractal-uri", type=str, default="localhost:7777", help="FractalServer location to pull from")
```

Comment:
Do you want to change the default from localhost here as well?

Reply:
No, QCPortal is user facing. Anything here will be developer facing and compute cannot be arbitrarily hooked up to our central server.